src/main/java/org/jenkinsci/plugins/durabletask/PowershellScript.java

```diff
@@ -48,6 +48,7 @@
 public final class PowershellScript extends FileMonitoringTask {
     private final String script;
     private String powershellBinary = "powershell";
+    private boolean loadProfile = false;
```

Comment:
false is the default
```suggestion
    private boolean loadProfile;
```